client/src/containers/Setting/Setting.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+
+class Setting extends Component {
+
+    constructor() {
+        super();
+        this.state = {
+            catLanguage:'En'
```

Comment:
don't use variable names like that! Don't be lazy with variable names, even if you're just testing stuff. It should be `categoryLanguage`

Reply:
you actually don't need that local state at all - you're not using state.catLanguage anywhere